WORKSPACE

```diff
@@ -164,10 +164,10 @@ scalatest_toolchain()
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "099a9fb96a376ccbbb7d291ed4ecbdfd42f6bc822ab77ae6f1b5cb9e914e94fa",
+    sha256 = "1105a42cfbaf12dba09be6aa3a9c340db1e5b655b07be5ac9cded1ee9ffc0f08",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip",
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip",
+        "https://storage.googleapis.com/engflow-tools-public/79e00373e1a9dc6363ae278a73264b39da44ad55.zip",
+        "https://github.com/bazelbuild/rules_go/archive/79e00373e1a9dc6363ae278a73264b39da44ad55.zip",
```

Comment:
Same here: the release `v0.36.0` points to this same commit, so there's no reason not to use `v0.36.0`.

(I remember I advised against using tags for `git_repository` since they may change. That's only important for `git_repository`. Our goal is to ensure we fetch the same bits every time or get an error if something changes. In `http_archive`, the `sha256` is used to check that. When `git_repository` is used with `tag` or `branch`, we have no guarantee that the repository owner did not change something. `commit` is better there, since it's hard to forge a SHA-1 hash).